MvvmCross.Forms.Wpf/Platforms/Wpf/Core/MvxFormsWindowsSetup.cs

```diff
@@ -15,14 +15,42 @@
 using MvvmCross.Platforms.Wpf.Core;
 using MvvmCross.Platforms.Wpf.Presenters;
 using MvvmCross.Platforms.Wpf.Views;
+using MvvmCross.Presenters;
 using MvvmCross.ViewModels;
 using Xamarin.Forms;
 
 namespace MvvmCross.Forms.Platforms.Wpf.Core
 {
-    public abstract class MvxFormsWpfSetup : MvxWpfSetup
+    public abstract class MvxFormsWpfSetup : MvxWpfSetup, IMvxFormsSetup
     {
         private Application _formsApplication;
+        private IMvxFormsSetupHelper _formsSetupHelper;
+
+        public virtual IMvxFormsSetupHelper FormsSetupHelper
+        {
+            get
+            {
+                return _formsSetupHelper ?? (_formsSetupHelper = Mvx.Resolve<IMvxFormsSetupHelper>());
+            }
+        }
+
+        protected override void RegisterImplementations()
+        {
+            base.RegisterImplementations();
+            Mvx.RegisterSingleton<IMvxFormsSetup>(this);
+
+            Mvx.LazyConstructAndRegisterSingleton<IMvxViewPresenter, MvxFormsWpfViewPresenter>();
+            Mvx.LazyConstructAndRegisterSingleton<IMvxFormsSetupHelper, MvxFormsSetupHelper>();
+            Mvx.Resolve<IMvxFormsSetupHelper>().InitializeIoC();
```

Comment:
If `IMvxFormsSetupHelper` is going to be used on the next line, maybe `ConstructAndRegisterSingleton` should be used instead of the lazy version (it also returns the instance back)